index.js

```diff
@@ -52,6 +52,12 @@ module.exports.createSlackMessage = (build) => {
           title: 'Status',
           value: build.status
         }, {
+          title: 'Repository',
```

Comment:
Could you make these optional by first checking if they actually exist? (like the `images` below)

Because there will be there only if it's a triggered build.